fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/EcamMessages/AbnormalSensed/ata80-rest.ts

```diff
@@ -122,13 +148,38 @@ export const EcamAbnormalSensedAta80Rest: { [n: number]: AbnormalProcedure } = {
     ],
   },
   990900010: {
-    title: '\x1b<4m\x1b4mMISC\x1bm SEVERE TURBULENCE (WIP)',
+    title: '\x1b<4m\x1b4mMISC\x1bm SEVERE TURBULENCE',
     sensed: false,
-    items: [], // TODO
+    items: [
+      { name: 'SEAT BELTS', sensed: true, labelNotCompleted: 'ON' },
+      { name: 'MAX TURB SPEED : 300 / .85 ', sensed: false },
+      { name: 'MIN TURB SPEED : GREEN DOT', sensed: false },
+      { name: 'ADJUST SPEED AS NECCESSARY FOR COMFORT', sensed: false },
+      { name: 'AP : KEEP ON', sensed: false },
+      { name: 'CABIN & CKPT (LOOSE EQPT)', sensed: false, labelNotCompleted: 'SECURE' },
+      { name: 'SPEED BRAKES', sensed: false, labelNotCompleted: 'AS RQRD' },
+      { name: 'IF EXCESSIVE THRUST VARIATIONS', sensed: false, condition: true },
```

Comment:
```suggestion
      { name: 'EXCESSIVE THRUST VARIATIONS', sensed: false, condition: true },
```
also here, IF not required